.circleci/config.yml

```diff
@@ -45,7 +39,6 @@ jobs:
 workflows:
   pr-build-test:
     jobs:
-      - build-1-19
```

Comment:
Why are we dropping testing on Go 1.19, when only now we're bumping the Go version in the go.mod files to 1.19?

Reply:
One of the library dependences is referencing `errors.Join` which apparently didn't exist until Go 1.20, so 1.19 doesn't pass anymore.  But I probably should have updated the go.mod version to 1.20, whoops.